src/Scorecard.ts

```diff
@@ -0,0 +1,56 @@
+import {CategoryScores} from "./domain";
+
+const ONES_THROUGH_SIXES_BONUS_SCORE = 63;
+
+export class Scorecard {
+  private chosenCategories: string[]
+  private scores: CategoryScores;
+  private yahtzeeBonus: boolean;
+
+  constructor() {
+    this.yahtzeeBonus = false;
+    this.chosenCategories = []
```

Comment:
What's the justification for a separate chosenCategories value? Could this not be calculated from `this.scores`? You'd have to have a different way of representing the difference between an unchosen 0 and a chosen 0 but it seems like it could condense this a bit